classes/order/OrderCarrier.php

```diff
@@ -72,9 +72,84 @@ class OrderCarrierCore extends ObjectModel
     );
 
     protected $webserviceParameters = array(
+        'objectMethods' => array('update' => 'updateWs'),
         'fields' => array(
             'id_order' => array('xlink_resource' => 'orders'),
             'id_carrier' => array('xlink_resource' => 'carriers'),
         ),
     );
+
+    /**
+     * @param Order $order Required
+     * @return bool
+     */
+    public function sendInTransitEmail($order)
+    {
+        $customer = new Customer((int)$order->id_customer);
+        $carrier = new Carrier((int)$order->id_carrier, $order->id_lang);
+        if (!Validate::isLoadedObject($customer)) {
+            throw new PrestaShopException('Can\'t load Customer object');
+        }
+        if (!Validate::isLoadedObject($carrier)) {
+            throw new PrestaShopException('Can\'t load Carrier object');
+        }
+        $orderLanguage = new Language((int) $order->id_lang);
+        $templateVars = array(
+            '{followup}' => str_replace('@', $order->shipping_number, $carrier->url),
+            '{firstname}' => $customer->firstname,
+            '{lastname}' => $customer->lastname,
+            '{id_order}' => $order->id,
+            '{shipping_number}' => $this->tracking_number,
```

Comment:
because shipping_number on Order is deprecated:
`@deprecated 1.5.0.4
@see OrderCarrier->tracking_number
`